### PR TITLE
OSDOCS-2708 - RN Add new feature SRO as tech preview

### PR DESCRIPTION
**Pull request by kmccarron-rh**
Jira for Release Note: https://issues.redhat.com/browse/OSDOCS-2708
The release note is dependent on:
 Jira for SRO feature addition: https://issues.redhat.com/browse/OSDOCS-2396

The Driver Toolkit is added to the Table listing Tech Preview features for 4.9. Here is the Driver Toolkit PR that added the TP feature to the 4.8 RNs: https://github.com/openshift/openshift-docs/pull/35249.  

Direct link to text addition: https://deploy-preview-36465--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#special-resource-operator-technology-preview

Table listing Tech Preview features:
https://deploy-preview-36465--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4-9-technology-preview

Release Note for 4.9

### CODE REVIEW

**Comment by kmccarron-rh** [comment]
@dagrayvid and @wabouhamad  - PTAL - Release Note for SRO (Tech Preview). 
Note: The link to the section in the documentation will be broken until the feature doc is merged.

**Comment by dagrayvid** [comment]
Looks good to me!

**Comment by kmccarron-rh** [comment]
Moving SRO feature text changes to a separate PR.

**Comment by kmccarron-rh** [comment]
Hi @wabouhamad! This change now ready,  PTAL.

**Review by wabouhamad — COMMENTED** [review]
/lgtm

**Comment by wabouhamad** [comment]
@kmccarron-rh looks good to me

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -596,6 +596,11 @@ The Metering Operator is deprecated as of {product-title} 4.6, and is scheduled
 [id="ocp-4-9-scale"]
 === Scale
 
+==== Special Resource Operator (Technology Preview)
```
Comment (ctauchen) [inline release_notes/ocp-4-9-release-notes.adoc:599]: Remember to add a heading anchor: perhaps `[id="ocp-4-9-scale-special-resource-operator"]
`

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -1515,6 +1520,16 @@ In the table below, features are marked with the following statuses:
 |-
 |TP
 
+|Driver Toolkit
+|-
+|TP
+|TP
```
Comment (ctauchen) [inline release_notes/ocp-4-9-release-notes.adoc:1526]: You didn't mention this in your PR comment. Can you explain?

Seems that this didn't get copied over from the 4.8 notes.

FYI @jeana-redhat

**Review by ctauchen — COMMENTED** [review]
Looks good! Just needs a heading anchor and we're all set.

Can you provide more information about the other TP feature you returned to the table?

File: release_notes/ocp-4-9-release-notes.adoc (hunk 2 above)
Comment (kmccarron-rh) [reply release_notes/ocp-4-9-release-notes.adoc:1526]: This is noted in the Release Note tracker for 4.9. I updated the initial comment in this PR to point to the Driver Toolkit PR that added it to the 4.8 RNs.

File: release_notes/ocp-4-9-release-notes.adoc (hunk 1 above)
Comment (kmccarron-rh) [reply release_notes/ocp-4-9-release-notes.adoc:600]: Done! Thanks for the reminder.

**Pull request merged**